codec-http/src/main/java/io/netty/handler/codec/http/websocketx/extensions/compression/WebSocketServerCompressionHandler.java

```diff
@@ -23,11 +24,10 @@
  *
  * See <tt>io.netty.example.http.websocketx.html5.WebSocketServer</tt> for usage.
  */
-public class WebSocketServerCompressionHandler extends WebSocketServerExtensionHandler {
+public final class WebSocketServerCompressionHandler extends WebSocketServerExtensionHandler {
+
+    public static final WebSocketServerCompressionHandler INSTANCE = new WebSocketServerCompressionHandler();
```

Comment:
What is the need for the ```INSTANCE``` member if it can't be reused and the constructor is public?